pkg/common/client/config.go

```diff
@@ -32,8 +33,9 @@ import (
 )
 
 type KafkaAuthConfig struct {
-	TLS  *KafkaTlsConfig
-	SASL *KafkaSaslConfig
+	Brokers string
+	TLS     *KafkaTlsConfig
```

Comment:
Pretty certain we don't want to do this.  Our long term direction is to put the brokers in the configmap instead of the secret (as the consolidated channel implementation does) and this is a temporary one-off for our own needs - let's try to find another workaround (or maybe just move the brokers if that's easier?)

Reply:
Since we're eventually planning to move the brokers out of the secret, I changed the GetAuthConfigFromKubernetes and GetAuthConfigFromSecret functions to return the broker string as part of the return tuple, but with a broker-free KafkaAuthConfig struct.  This should make it moderately easy to extricate and reposition later (though I would rather not do the move-to-configmap work as part of this particular issue).